lib/Util/Notices/PermanentNotices.php

```diff
@@ -61,10 +61,8 @@ public function init() {
     $this->inactive_subscribers_notice->init(
       Menu::isOnMailPoetAdminPage($exclude = ['mailpoet-welcome-wizard'])
     );
-    $this->discounts_announcement->init(
-      empty($_GET['page'])
-      && $this->wp->isAdmin()
-      && strpos($_SERVER['SCRIPT_NAME'], 'wp-admin/index.php') !== false
+    $this->black_friday_notice->init(
+      Menu::isOnMailPoetAdminPage($exclude = ['mailpoet-welcome-wizard'])
```

Comment:
I think we might want to exclude also pages
`mailpoet-woocommerce-list-import` and `mailpoet-revenue-tracking-permission` possibly also for older notices. These two are displayed to users with WooCommerce right after the wizard.